pilz_robot_programming/test/integrationtests/tst_api_cmd_conversion.py

```diff
@@ -169,6 +172,42 @@ def test_ptp_cmd_convert_pose(self):
         self._analyze_request_general(PLANNING_GROUP_NAME, "PTP", EXP_VEL_SCALE, EXP_ACC_SCALE, req)
         self._analyze_request_pose(TARGET_LINK_NAME, exp_goal_pose, req)
 
+    def test_ptp_cmd_convert_stamped_pose(self):
+        """ Check that conversion to MotionPlanRequest works correctly.
```

Comment:
Please state here explicitly that setting a timestamp does not work as one may expect.